.github/workflows/twister.yaml

```diff
@@ -15,6 +15,9 @@ on:
     # Run at 17:00 UTC on every Saturday
     - cron: '0 17 * * 6'
 
+permissions:
+  contents: read
+
```

Comment:
shouldn;t this action have `checks: write` permissions in 'Publish Unit Test Results' section below?

Reply:
yes! thanks for paying attention